src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -95,6 +95,10 @@ public DataSet updateNeedsReviewDataSets(Integer dataSetId, Boolean needsApprova
         return datasetDAO.findNeedsApprovalDataSetByDataSetId(dataSetIdList);
     }
 
+    public Set<DatasetDTO> findDatasetsByDacIds(List<Integer> dacIds) {
+        return datasetDAO.findDatasetsByDacIds(dacIds);
```

Comment:
Minor safety check on an empty id list might be useful here - that way we'll be sure never to send the DAO call an empty list.